src/main/java/mServer/crawler/sender/arte/ArteJsonObjectToDatenFilmCallable.java

```diff
@@ -52,52 +55,92 @@ public ArteJsonObjectToDatenFilmCallable(JsonObject aJsonObjec, String aLangCode
     @Override
     public DatenFilm call() {
         DatenFilm film = null;
-        if(jsonObject != null && jsonObject.has(JSON_OBJECT_KEY_PROGRAM))
+        try {
+        if(isValidProgramObject(jsonObject))
         {
-            JsonObject programObject = jsonObject.get(JSON_OBJECT_KEY_PROGRAM).getAsJsonObject();
-            if(isValidProgramObject(programObject))
+            String titel = getTitle(jsonObject);
+            String thema = getSubject(jsonObject);
+
+            String beschreibung = getElementValue(jsonObject, JSON_ELEMENT_KEY_SHORT_DESCRIPTION);
+
+            String urlWeb = getElementValue(jsonObject, JSON_ELEMENT_KEY_URL);
+
+            //https://api.arte.tv/api/player/v1/config/[language:de/fr]/[programId]?platform=ARTE_NEXT
+            String programId = getElementValue(jsonObject, JSON_ELEMENT_KEY_PROGRAM_ID);
+            String videosUrl = String.format(ARTE_VIDEO_INFORMATION_URL_PATTERN, langCode, programId);
+
+            Gson gson = new GsonBuilder().registerTypeAdapter(ArteVideoDTO.class, new ArteVideoDeserializer()).create();
+
+            try(Response responseVideoDetails = executeRequest(videosUrl))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)